smaug/python/ops/activation_ops.py

```diff
@@ -69,7 +52,7 @@ def get_activation_op(activation):
   Args:
     activation: A string representing the activation function.
   """
-  op_type = _get_activation_type(activation)
+  op_type = _activation_types[activation]
```

Comment:
Erm, wait, if the goal is to get a function pointer, why not just store function pointers in the dict directly, alongside the enum?